src/Databases/DatabaseOnDisk.cpp

```diff
@@ -690,6 +690,17 @@ ASTPtr DatabaseOnDisk::getCreateQueryFromMetadata(const String & database_metada
     if (ast)
     {
         auto & ast_create_query = ast->as<ASTCreateQuery &>();
+        if (ast_create_query.storage &&
+            ast_create_query.storage->engine->name == "Kafka")
+        {
+          for (auto &it : ast_create_query.storage->settings->changes)
+          {
+            if (it.name == "kafka_sasl_password")
+            {
+              it.value.assignString("xxxxx",5);
+            }
+          }
```

Comment:
Sorry, we can not accept that kludge. That kind of tight coupling is not acceptable.

You can just remove that (that it will create security concerns), or think of something more SOLID.